tools/releases/dockerfiles/Dockerfile.kuma-cp

```diff
@@ -5,11 +5,16 @@ ADD $KUMA_ROOT/build/artifacts-linux-amd64/kuma-cp/kuma-cp /usr/bin
 RUN mkdir -p /etc/kuma
 ADD $KUMA_ROOT/pkg/config/app/kuma-cp/kuma-cp.defaults.yaml /etc/kuma
 
-RUN mkdir /kuma
-COPY $KUMA_ROOT/tools/releases/templates/LICENSE /kuma
-COPY $KUMA_ROOT/tools/releases/templates/NOTICE /kuma
-COPY $KUMA_ROOT/tools/releases/templates/README /kuma
+COPY $KUMA_ROOT/tools/releases/templates/LICENSE \
+    $KUMA_ROOT/tools/releases/templates/README \
+    /kuma/
 
-USER nobody:nobody
+COPY $KUMA_ROOT/tools/releases/templates/NOTICE /kuma/
```

Comment:
Why not bundle also this in the previous `COPY`?

Reply:
This way, all the images will share a common layer for LICENSE&README, and only diverge on the NOTICE. It was intended as a micro-optimization, but thinking about it again, it's really not useful. Only kumactl diverges.

Will fix.